conf/defaults.ini

```diff
@@ -231,6 +231,7 @@ verify_email_enabled = false
 
 # Background text for the user field on the login page
 login_hint = email or username
+passowrd_hint = password
```

Comment:
this should be added to `sample.ini` and `docs/configuration.md` as well with a comment explaining what it does.

Reply:
and spelled correctly? :)